sdk/src/Services/DynamoDBv2/Custom/DataModel/Context.cs

```diff
@@ -53,6 +53,61 @@ public partial class DynamoDBContext : IDynamoDBContext
 
         #endregion
 
+        #region Public methods
+        /// <summary>
+        /// Adds a <see cref="Table"/> to this context's internal cache, which
+        /// will avoid the need to fetch table metadata automatically from DynamoDB.
+        /// This may be used in conjunction with an <see cref="ITableBuilder"/>.
+        /// </summary>
+        /// <remarks>
+        /// Using this method can avoid latency and potential deadlocks due to the internal 
```

Comment:
Instead of `potential deadlocks` can we say `thread starvation due to blocking asynchronous <see cref="IAmazonDynamoDB.DescribeTable(string)"/> calls ...`. To me it looks like if we just say `potential deadlocks` as a reader I would say AWS fix should fix deadlocks not realizing this is the fix.

Reply:
Fixed in https://github.com/aws/aws-sdk-net/compare/74020f5167a3bf2e54dd53bdeba482f965e43dd2..abcc18e865203cd0aa124c2c175f0dc798d5be93.